lm_eval/tasks/gpqa/cot_n_shot/_gpqa_cot_n_shot_yaml

```diff
@@ -0,0 +1,30 @@
+dataset_path: Idavidrein/gpqa
+group: gpqa
+output_type: generate_until
+process_docs: !function utils.process_docs
+training_split: train
+# Because huggingface dataset only has train split
+validation_split: train
+test_split: null
+description: "Here are some example questions from experts. Answer the final question yourself, following the format of the previous questions exactly.\n"
+doc_to_text: "Question: {{Question}}\nChoices:\n(A) {{choice1}}\n(B) {{choice2}}\n(C) {{choice3}}\n(D) {{choice4}}\nLet's think step by step: "
+doc_to_target: answer
+filter_list:
+  - name: "get-answer"
```

Comment:
Let's have all the GPQA tasks have both a `strict-match` and `flexible-extract` filter pipeline: so `get-answer` should have its name changed to `strict-match` and we should ensure all variants (including CoT n-shot and generative_n_shot) have both the strict and flexible filters.